prac_06/guitar_test.py

```diff
@@ -0,0 +1,13 @@
+from guitar import Guitar
+
+# Create Guitar instances
+guitar1 = Guitar("Gibson L-5 CES", 1922, 16035.40)
+guitar2 = Guitar("Another Guitar", 2013)
+
+# Test get_age() method
+print(f"{guitar1.name} get_age() - Expected 100. Got {guitar1.get_age()}")
+print(f"{guitar2.name} get_age() - Expected 9. Got {guitar2.get_age()}")
+
+# Test is_vintage() method
+print(f"{guitar1.name} is_vintage() - Expected True. Got {guitar1.is_vintage()}")
+print(f"{guitar2.name} is_vintage() - Expected False. Got {guitar2.is_vintage()}")
```

Comment:
Use assert statements like assert guitar1.get_age() == 100 for automated test validation.